### PR TITLE
Support AES Key Wrap for CMS key encryption

### PR DESCRIPTION
**Pull request by eukaryote**
- adds KeyEncryptionAlgorithmId oids for aes key wrap (rfc3394)

### CODE REVIEW

File: asn1crypto/cms.py [hunk 1]
```diff
@@ -419,7 +419,10 @@ class RecipientIdentifier(Choice):
 
 class KeyEncryptionAlgorithmId(ObjectIdentifier):
     _map = {
-        '1.2.840.113549.1.1.1': 'rsa'
+        '1.2.840.113549.1.1.1': 'rsa',
+        '2.16.840.1.101.3.4.1.5': 'aes128-wrap',
+        '2.16.840.1.101.3.4.1.25': 'aes192-wrap',
+        '2.16.840.1.101.3.4.1.45': 'aes256-wrap',
     }
```
Comment (wbond) [inline asn1crypto/cms.py]: For consistency with the other `asn1crypto` OID names, can we use `aes128_wrap`, etc?

**Comment by eukaryote** [comment]
Yeah, of course. I was mindlessly copying form the NIST doc and didn't think to check the convention. 

I didn't realize the second commit I made and was going to make a separate pull request for would get merged into this one. I'll figure out how to remove the 2nd set of commits and will modify the names to use the underscore convention.

**Comment by eukaryote** [comment]
Updated to use underscores, and I also added three other OIDs for the padding variants of aes key wrap.

**Comment by wbond** [comment]
Awesome, thanks for the contribution!

**Pull request merged**